README.md

```diff
@@ -324,6 +322,21 @@ Depending on your needs, at this you might find it useful to continue with learn
 
 [Learn more about PDFSlick](https://pdfslick.dev) | [Checkout the Examples](https://pdfslick.dev/examples)
 
+## How to join us
+```
+npm install;
+npm run dev;
+```
+
+- Open http://localhost:3000 for React website examples (Nextjs server).
+- Open http://localhost:5174/ for Svelte website examples.
+- Open http://localhost:3004/ for Solid website examples.
+- Copy ./assets/pdfs/* to
+  + ./apps/solidweb/public/pdfs/
+  + ./apps/svelteweb/public/pdfs/
+  + ./apps/web/public/pdfs/
+  + ./apps/vanillajs/public/pdfs/
+
```

Comment:
Let's remove this, will add a separate local dev/contribution section including these and similar details.